Makefile

```diff
@@ -89,9 +89,8 @@ doc-frontend: install-frontend ## Frontend src doc
 
 # Install
 
-
 .PHONY: install
-install: install-pre-commit install-backend install-mdbook install-frontend install-e2e install-types  ## Install ALL dependencies within the source repo
+install: install-pre-commit install-backend install-mdbook install-frontend  ## Install ALL dependencies within the source repo
```

Comment:
probably create subtargets like `install-most` if `make install` fails for you? With this proposed change, the comment would be wrong.

Reply:
We'll revert this.